content/200-concepts/100-components/01-prisma-schema/06-relations/300-many-to-many-relations.mdx

```diff
@@ -14,14 +14,14 @@ Prisma schema syntax and the implementation in the underlying database differs b
 
 ## Relational databases
 
-In relational databases, m-n-relations are typically modelled via [relation tables](/concepts/components/prisma-schema/relations/many-to-many-relations#relation-tables). m-n-relations can be either [explicit](#explicit-many-to-many-relations) or [implicit](#implicit-many-to-many-relations) in the Prisma schema.
+In relational databases, m-n relations are typically modelled via [relation tables](/concepts/components/prisma-schema/relations/many-to-many-relations#relation-tables). m-n relations can be either [explicit](#explicit-many-to-many-relations) or [implicit](#implicit-many-to-many-relations) in the Prisma schema. We recommend using [implicit](#implicit-many-to-many-relations) m-n relations if you do not need to store any additional meta-data in the relation table itself. You can always migrate to an [explicit](#explict-many-to-many-relations) m-n relation later if needed.
 
 ### Explicit many-to-many relations
 
-In an explicit many-to-many relation, the **relation table is represented as a model in the Prisma schema** and can be used in queries. Explicit many-to-many relations define three models:
+In an explicit m-n relation, the **relation table is represented as a model in the Prisma schema** and can be used in queries. Explicit m-n relations define three models:
```

Comment:
Change in our style guide? I remember not long ago this was expanded.

Reply:
wanted to keep consistency as in the section heading we [mention](https://www.prisma.io/docs/concepts/components/prisma-schema/relations/many-to-many-relations#relational-databases) m-n relations